assets/js/googlesitekit/widgets/register-defaults.js

```diff
@@ -103,10 +104,7 @@ export function registerDefaults( widgetsAPI ) {
 					<KeyMetricsNewBadge />
 				</Fragment>
 			),
-			subtitle: __(
-				'Track progress towards your goals with tailored metrics',
-				'google-site-kit'
-			),
+			subtitle: MetricsWidgetSubtitle,
```

Comment:
I included subtitle as component, to allow for conditional rendering, since in new design, subtitle is show only after KMW is setup